soc/silabs/silabs_siwx91x/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 # Copyright (c) 2024 Silicon Laboratories Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+include(west)  # Add this at the top or before using ${WEST}
+
 add_subdirectory(siwg917)
+zephyr_include_directories(.)
```

Comment:
What is the purpose of changing the include path of the CMake build? This seems unrelated to signing support.